src/state_machine.cpp

```diff
@@ -655,9 +655,15 @@ void Base::handleReadCard() {
 
 void Idle::entry() {
   LOG(state_log, s_info, str_enter(), str_Idle());
+  tonuino.disableAmplifier();
```

Comment:
Maybe it would be better to do this in a base class depending on a flag in the subclass, but I haven't figured out how to do that yet. I'm just a Java programmer after all. ;-)